sale_order_partner_no_autofollow/models/sale_order.py

```diff
@@ -0,0 +1,69 @@
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def message_subscribe(self, partner_ids=None, channel_ids=None, subtype_ids=None):
+        if self.env.context.get("sale_partner_disable_autofollow", False):
+            partner_ids.remove(self.partner_id.id)
+        return super(SaleOrder, self).message_subscribe(
+            partner_ids, channel_ids, subtype_ids
+        )
+
+    @api.returns("mail.message", lambda value: value.id)
+    def message_post(self, **kwargs):
```

Comment:
Do you really need to override this method? It will call message_subscribe() to add partners anyway. Which is already overridden.